Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -7265,6 +7265,46 @@ template<CSSPropertyID property> RefPtr<CSSValue> consumeBackgroundSize(CSSParse
     return createPrimitiveValuePair(horizontal.releaseNonNull(), vertical.releaseNonNull(), identicalValueEncoding);
 }
 
+RefPtr<CSSValueList> consumeAlignTracks(CSSParserTokenRange& range)
+{    
+    RefPtr<CSSValueList> parsedValues = CSSValueList::createCommaSeparated();
+
+    do {
+        if (range.atEnd())
+            break;
+
+        auto value = consumeContentDistributionOverflowPosition(range, isContentPositionOrLeftOrRightKeyword);
```

Comment:
```suggestion
        auto value = consumeContentDistributionOverflowPosition(range, isContentPositionKeyword);
```

seems to be correct according to the syntax and your test